### DISCUSSION TITLE
Overclocking

### DISCUSSION THREAD

**Discussion by NewsGuyTor** [OnionUI/Onion#755] [Ideas]
GarlicOS has easy overclocking, that would be awesome in Onion! 

Since it's the same chip I don't think it should be too hard to implement for OnionOS.

**Comment by Aemiii91** [comment]
Yes, overclocking is a planned feature. The Miyoo Mini's CPU (SigmaStar SSD202D) is not the same as on RG35XX (Actions ATM7039S).

**Reply by NewsGuyTor** [reply]
Awesome, looking forward to it!

**Comment by schmurtzm** [comment]
Already available for Retroarch, more information here : https://github.com/OnionUI/Onion/issues/768
It's also used with Netplay (see [here](https://onionui.github.io/docs/multiplayer#tips-)).
We have plan to extend it from any Onion emulator so I keep it open for now.
